src/EditorFeatures/CSharpTest/ExtractClass/ExtractClassTests.cs

```diff
@@ -2230,6 +2231,65 @@ public T3 Method()
             }.RunAsync();
         }
 
+        [Fact]
+        public async Task TestIncompleteFieldSelection_NoAction()
+        {
+            var input = @"
+class C
+{
+    pub[||] int Foo = 0;
+}
+";
+            var test = new Test
+            {
+                TestCode = input,
+                FixedCode = input
+            };
+            // no need for dialog selection because we shouldn't activate at all
+            test.ExpectedDiagnostics.Add( DiagnosticResult.CompilerError("CS1519").WithSpan(4, 9, 4, 12).WithArguments("int"));
```

Comment:
```suggestion
            var input = @"
class C
{
    {|CS1519:pub|}[||] int Foo = 0;
}
";
            var test = new Test
            {
                TestCode = input,
                FixedCode = input
            };
            // no need for dialog selection because we shouldn't activate at all
```